go.mod

```diff
@@ -0,0 +1,25 @@
+module mattermost-plugin-aws-SNS
```

Comment:
thanks for moving to use go modules, can you remove the `Gopkg.toml` file as well?

Reply:
To be honest, I am a _go_ beginner and I have no clue regarding packaging of go apps/libs. Therefore I would like to ask you to take care of this afterwards. Probably you can remove the whole `vendor` directory from the repository as it should be handled by the module dependencies, right?

Reply:
yes, and then need to check if the build will continue pass, maybe some changes in the circleci will be needed as well. can look this: https://github.com/mattermost/mattermost-plugin-sample and make the same changes the upstream have. I also can do that when i have some time